Examples/AblyPush/README.md

```diff
@@ -1,12 +1,26 @@
+## Push Notifications Example
+
 This app gives you a brief understanding of how Ably's Push Notifications work. 
 You will need a real iOS device to test this functionality.
 
 - First, go to https://ably.com/accounts/, create an app and copy API key from API Keys tab of the app's dashboard.
 - Go to the `Notifications` tab of the Ably's app dashboard and scroll to the `Push Notifications Setup` section. Press `Configure Push` and follow the instructions there.
 - Insert it instead of an empty string in the `key` property's value inside the `AblyHelper` class.
-- Make sure you've selected your development team in the `Signing & Capabilities` tab of the Xcode project target settings and all the provisioning profiles created without errors.
+- Make sure you've selected your development team in the `Signing & Capabilities` tab of the Xcode project target settings and all the provisioning profiles created without errors (update the `bundle-id` for the `AblyPushExample` target as needed).
 - Build and Run the app on your device.
 - Hit "Activate" button, then hit "Print Token". You should see "IDENTITY TOKEN: exists" printed in the debug output window.
 - Hit "Device Details" to display Ably push device registration info after successful activation.
 - Go to the `Notifications` tab of the Ably's app dashboard again and scroll to the `Push Inspector` section. Fill in `Title` and `Body` fields of the push notification. Then insert "basic-apns-example" in the `Client ID` field and press enter. Then hit "Push to client" button. You should now see the notification on the screen of your device.
-- Also you can send notifications from the app itself if you tick `Push Admin` capability in your API key settings in the app's dashboard. Just hit "Send Push" button to send a predefined push notification which is also will be displayed on your device's screen right away. You can change this behavior in the `UNUserNotificationCenterDelegate` extension for the `AblyHelper` by editing `completionHandler([.banner, .sound])` line of code.
\ No newline at end of file
+- Also you can send notifications from the app itself if you tick `Push Admin` capability in your API key settings in the app's dashboard. Just hit "Send Push" button to send a predefined push notification which is also will be displayed on your device's screen right away. You can change this behavior in the `UNUserNotificationCenterDelegate` extension for the `AblyHelper` by editing `completionHandler([.banner, .sound])` line of code.
+
+### Location pushes
+
+* In order to use this capability (starting from iOS 15), you need to apply for the special entitlement on the Apple's developer portal. Follow instructions [here](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_location_push) on how to do that.
+
+* Update the `bundle-id` for the `AblyPushExample` and `AblyLocationPush` targets as needed.
```

Comment:
What about configuring the security group identifier in the code? And, is there a way that we can perhaps make it a single string that they configure as a build setting in Xcode, which then gets inserted into all the right places (i.e. the code and the entitlements file)?

Reply:
https://github.com/ably/ably-cocoa/pull/1771/commits/fd06242e3d0e3432ac91521fbd9f15ca831242d8